### PR TITLE
Fixing small typo in docs

### PR DESCRIPTION
**Pull request by gunnaraas**
'environment' was spelled 'environmemt' on line 649

### CODE REVIEW

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -646,7 +646,7 @@ confirm your paths, for example::
 If any of your system environment variables are not present in the file then
 you must add them.
 
-Alternatively it is possible to use the environemt variables of your system by modifying
+Alternatively it is possible to use the environent variables of your system by modifying
```
Comment (skjnldsv) [inline admin_manual/installation/source_installation.rst]:
```suggestion
Alternatively it is possible to use the environment variables of your system by modifying
```
:grin:

**Comment by MorrisJobke** [comment]
Master version is in #1992 (back ports are there too)

**Pull request merged**